server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const userSchema = new mongoose.Schema({
+    user:{
+        type: mongoose.Schema.Types.ObjectId,
+        ref:'user'
+    },
+    name: [
+        {
+    first_name: {
+        type: String,
+        required: true
+      },
+      last_name: {
+        type: String,
+        required: true
+      }
+    }
+],
+phone: {
+    type: String,
+    required: false
+  },
+  home_address: {
+    type: String,
+    required: true
+  },
+    description: {
+    type: String,
+    required: true
+  },
+  register_date: {
+    type: Date,
+    default: Date.now
+  }
+
+    
+});
```

Comment:
```suggestion
}, { timestamps: true });
```
You can pass this field as a second object to Schema. This creates two fields `createdAt` and `updatedAt`. So, we don't need `register_date` field.